extensions/classifiers/text-input.tokenizer.spec.ts

```diff
@@ -18,24 +18,18 @@
 
 // TODO(#7222): Remove the following block of unnnecessary imports once
 // the code corresponding to the spec is upgraded to Angular 8.
```

Comment:
What about these comments, should they be removed?

Reply:
Yes, they should be removed.
Done.